src/main/java/javax/validation/ConstraintViolation.java

```diff
@@ -61,14 +61,17 @@
 	 *     <li>the bean instance the constraint is applied on if it is
 	 *     a bean constraint</li>
 	 *     <li>the bean instance hosting the property the constraint
-	 *     is applied on if it is a property constraint</li>
+	 *     is applied on if it is a property constraint or a container-element constraint
```

Comment:
We usually don't use the hyphen in the spec. Would have fixed it right away but I thought it might be better to be sure?